### PR TITLE
Move all non-dynamic styling into CSS and made miscellaneous style changes

### PR DESCRIPTION
**Pull request by connorbo97**
- Moves React in-line stylings that are not dynamic into CSS files
- Fixes issue with panel 'crunching' when you open and close it
- When closing/opening panel, dropdown no longer shifts left and right
- Added margin to Codemirrror-lines to ensure cursor doesn't get cut off
- Made style consistent between Login.js and CreateUser.js
     -Removed CreateUser.css, as now CreateUser uses Login.css
- Refactored/Split source doc strings into functions in Output.js

### CODE REVIEW

File: src/components/CreateUser.js [hunk 1]
```diff
@@ -133,156 +139,95 @@ class CreateUser extends React.Component {
...
+          waiting={this.state.waiting}
+          onChange={this.updateUsername}
+        />
+        <LoginInput
+          type={"password"}
+          data={this.state.password}
+          waiting={this.state.waiting}
+          onChange={this.updatePassword}
+        />
+        {this.renderErrorMessage(this.state.passwordMessage)}
```
Comment (malsf21) [inline src/components/CreateUser.js]: @connorbo97 should we explicitly do `this.renderErrorMessage(this.state.passwordMessage, false)`?

File: src/components/CreateUser.js [hunk 2]
```diff
@@ -133,156 +139,95 @@ class CreateUser extends React.Component {
...
-              )}
-              {this.renderButton()}
-              <Link to="/login" className="create-form-link">
-                Already have an account? Click here to log in
-              </Link>
-            </div>
-          </div>
+      <div className="login-modal">
+        <form className="login-form" onSubmit={this.submit}>
+          {" "}
```
Comment (malsf21) [inline src/components/CreateUser.js]: Is there a reason that this is here? Is it the modal hack thing? @connorbo97

Comment (connorbo97) [reply src/components/CreateUser.js]: Yup, modal hack. It's probably a good idea to investigate why its needed/remove it. I'll make a card in trello bout it.

File: src/components/CreateUser.js (hunk 1 above)
Comment (malsf21) [reply src/components/CreateUser.js]: Upon further consideration, this is fine with the specialized scope of the function.

**Pull request merged**